Youtube clone - Add embeded images and video

kevcap opened the pull request:
Youtube project is done with requested features.

fdfragoso committed:
Add main elements

Embed video;
Description and status from embed video.

fdfragoso committed:
Added channel details

kevcap committed:
Create img folder and add pic1.jpg

kevcap committed:
Finished up the sidebar (HTML & CSS) and updated some parts of index.html

- Added some CSS and thumbnails for videos recommendations;
- Font colors and sizes
- I've created divs for put the main video reactions, inside it
- Changed the title of the page

fdfragoso committed:
Add more details to channel area

kevcap committed:
Add some videos to the sidebar and fixed some bugs (description)

Bugs:
-Youtube's logo letters "b" and "e" was transparent;
- User's avatar was all white, with no shape;
- Add a class "show-more" to put the cursor as pointer;

kevcap committed:
deleted img folder

fdfragoso committed:
Add README and screenshot img

fdfragoso committed:
Update README.md

New link for live version

fdfragoso committed:
Update README.md

Updated the display for social medias

davitomix commented:
### Pull Request under review by **[TSE] David Elí.**

davitomix commented:
### **Status: Changes required.** 🛑 👀 🔍 
### Hi Team. 😄 
### Very good job so far, but we have a couple of issues to solve before move on. 💯 👍 

### Implementation details:
**Comments div or section:**

- [ ]  Implement the comments section. Here's an example or you can follow the example from [Pathwright](https://www.youtube.com/watch?v=V74l_zS1x8E).

![Screen Shot 2020-02-20 at 12 22 03 PM](https://user-images.githubusercontent.com/52430951/74966023-ebf70880-53db-11ea-85d8-1487679ce8dc.png)

You can add a couple of comments there's no need to be too many.

****

**HTML Validator:**

- [ ] The validator drops many warnings an errors please solve them.
Heres the link of the [validator](https://validator.w3.org/#validate_by_input). 

![Screen Shot 2020-02-20 at 12 26 34 PM](https://user-images.githubusercontent.com/52430951/74966223-4bedaf00-53dc-11ea-8162-532c84ab84a6.png)

****

Please make the changes required and submit a new code review once you are ready! 🙏 

Happy coding! 😄 💻 

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)

kevcap committed:
Fixed some html validations (description)

-Removed "viewbox" and "perspectiveAspectRatio" of the <g> tags;
- Changed the child of the <button> from <div> to <span>;
- strayed the two </hr> that was within the code.
*Still facing a problem on line 211, cause the html validator says that there's no <p> (open), but have a </p>, when it its no true. The <p> is on line 207 and the </p> is on the line 211, closing the "p" scope

kevcap committed:
Update index.html

- fixing p tag at 287

kevcap committed:
Revert "Update index.html"

This reverts commit f84f497e2cb6fa7880871be1705f4dd1568cdd70.

fdfragoso committed:
Add youtube comment section

Still missing love icon from video's owner, reply and add more comments

kevcap committed:
Add heart reaction of channel owner

fdfragoso committed:
Update style.css

Heart red and white in the right position and size.

kevcap committed:
Add comments div and it's features

-Comments;
-Reactions;
-Avatar images;

SofiBretz commented:
### Well done Team! 👏
The design looks great! Good teamwork, keep it up! ✨.

Project Approved.

**You can merge 👍**

SofiBretz.
